opaque_keys/edx/tests/test_locators.py

```diff
@@ -59,3 +61,59 @@ def test_version_tree(self):
         test_id = ObjectId(test_id_loc)
         valid_locator = CourseLocator(version_guid=test_id)
         self.assertEqual(VersionTree(valid_locator).children, [])
+
+
+@ddt.ddt
+class CourseLocatorV2Tests(TestCase):
+    """
+    Tests for :class:`.CourseLocatorV2`
+    """
+
+    def test_course_locator_v2(self):
+        """
+        Verify that the method "from_string" of class "CourseKeyV2"
+        returns an object of "CourseLocatorV2" for a valid course key.
+        """
+        course_key_v2 = 'course-v2:org+course'
+        course_locator_v2 = CourseKeyV2.from_string(course_key_v2)
+        expected_course_locator = CourseLocatorV2(org='org', course='course')
+        self.assertEqual(expected_course_locator, course_locator_v2)
+
+    @ddt.data(
+        'org/course/run',
```

Comment:
There is no need for `ddt` in this instance.  Whether we use the old or new format for `CourseKey` doesn't matter. The method simply expects an instance of `CourseKey`.